<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionSelectDialog</class>
 <widget class="QDialog" name="SectionSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбрать раздел</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ICEGUI_NS::FilterTreeView" name="filterTreeView"/>
   </item>
   <item>
    <widget class="QLabel" name="descriptionL">
     <property name="text">
      <string>Описание:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionTE">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICEGUI_NS::FilterTreeView</class>
   <extends>QTreeView</extends>
   <header>filtertreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>okPB</sender>
   <signal>clicked()</signal>
   <receiver>SectionSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPB</sender>
   <signal>clicked()</signal>
   <receiver>SectionSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
